packages/ui/src/services/resubmitCourtCase.ts

```diff
@@ -1,10 +1,13 @@
 import type { AuditLogEvent } from "@moj-bichard7/common/types/AuditLogEvent"
+import EventCategory from "@moj-bichard7/common/types/EventCategory"
+import EventCode from "@moj-bichard7/common/types/EventCode"
+import getAuditLogEvent from "@moj-bichard7/core/lib/getAuditLogEvent"
 import serialiseToXml from "@moj-bichard7/core/lib/serialise/ahoXml/serialiseToXml"
 import type { AnnotatedHearingOutcome } from "@moj-bichard7/core/types/AnnotatedHearingOutcome"
+import { AUDIT_LOG_EVENT_SOURCE } from "config"
 import amendCourtCase from "services/amendCourtCase"
 import type User from "services/entities/User"
 import insertNotes from "services/insertNotes"
-import sendToQueue from "services/mq/sendToQueue"
```

Comment:
Why have you removed this? 

It has introduced more code in this file and others.

Reply:
function sendToQueue did not have any logic, it was just calling gateway.execute with messageXml and QueueName. Additionally, it created a new StompitMqGateway(config) instance with each function call, instead of having it injected as a dependency from the top level where it could be reused without creating new instances